src/modules/Telemetry/Sensor/BME680Sensor.cpp

```diff
@@ -1,18 +1,26 @@
 #include "BME680Sensor.h"
 #include "../mesh/generated/meshtastic/telemetry.pb.h"
+#include "FSCommon.h"
 #include "TelemetrySensor.h"
 #include "configuration.h"
-#include <Adafruit_BME680.h>
 
 BME680Sensor::BME680Sensor() : TelemetrySensor(meshtastic_TelemetrySensorType_BME680, "BME680") {}
 
 int32_t BME680Sensor::runOnce()
 {
-    LOG_INFO("Init sensor: %s\n", sensorName);
+    LOG_INFO("Init sensor: %s with the BSEC Library\n", sensorName);
     if (!hasSensor()) {
         return DEFAULT_SENSOR_MINIMUM_WAIT_TIME_BETWEEN_READS;
     }
-    status = bme680.begin(nodeTelemetrySensorsMap[sensorType]);
+    bme680.begin(nodeTelemetrySensorsMap[sensorType], Wire);
```

Comment:
How does the new I2C scan work if this is detected on the secondary bus? Right now all sensors assume 'Wire'